python/TestHarness/schedulers/Job.py

```diff
@@ -241,6 +242,8 @@ def getStatus(self):
         return self.status.getStatus()
     def getStatusMessage(self):
         return self.status.getStatusMessage()
+    def getStatusCode(self):
+        return self.status.getStatusCode()
     def setStatus(self, status, message=''):
```

Comment:
Who calls this method? It should probably be documented. This method shouldn't be used by most objects (including the TestHarness). Maybe the Scheduler uses it?